dev/PipsControl/PipsControl.xaml

```diff
@@ -0,0 +1,105 @@
+                            </VisualStateGroup>
+
+                            <VisualStateGroup x:Name="PreviousPageButtonIsEnabledStates">
+                                <VisualState x:Name="PreviousPageButtonEnabled"/>
+                                <VisualState x:Name="PreviousPageButtonDisabled">
+                                    <VisualState.Setters>
+                                        <Setter Target="PreviousPageButton.IsEnabled" Value="False"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                            </VisualStateGroup>
+
+                            <VisualStateGroup x:Name="NextPageButtonVisibilityStates">
+                                <VisualState x:Name="NextPageButtonHidden"/>
+                                <VisualState x:Name="NextPageButtonVisible">
+                                    <VisualState.Setters>
+                                        <Setter Target="NextPageButton.Opacity" Value="100"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                                <VisualState x:Name="NextPageButtonCollapsed">
+                                    <VisualState.Setters>
+                                        <Setter Target="NextPageButton.Visibility" Value="Collapsed"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                            </VisualStateGroup>
+
+                            <VisualStateGroup x:Name="NextPageButtonIsEnabledStates">
+                                <VisualState x:Name="NextPageButtonEnabled"/>
+                                <VisualState x:Name="NextPageButtonDisabled">
+                                    <VisualState.Setters>
+                                        <Setter Target="NextPageButton.IsEnabled" Value="False"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                            </VisualStateGroup>
+                        </VisualStateManager.VisualStateGroups>
+
+                        <Button x:Name="PreviousPageButton" Style="{TemplateBinding PreviousButtonStyle}"/>
+
+                        <ScrollViewer x:Name="VerticalPipsScrollViewer"
+                                      Grid.Row="1"
+                                      VerticalScrollBarVisibility="Hidden"
+                                      VerticalScrollMode="Disabled">
+                            <muxc:ItemsRepeater x:Name="VerticalPipsItemsRepeater"
+                                ItemsSource="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.VerticalPipsItems}">
+                                <muxc:ItemsRepeater.ItemTemplate>
+                                    <DataTemplate>
+                                        <Button Tag="{Binding}"/>
```

Comment:
>Tag [](start = 48, length = 3)

I don't think we should be using the tag property, and I don't think we have to since the repeater index matches the page index